src/app/component/Login.tsx

```diff
@@ -1,11 +1,44 @@
-import React from 'react'
+
+import React, { FormEvent, useState } from 'react'
 import 'bootstrap/dist/css/bootstrap.css'
 import style from '../page.module.css'
 import {BsPersonCircle} from 'react-icons/bs'
+import axios from 'axios'
+import { useMutation } from '@tanstack/react-query'
+import LoggedIn from './LoggedIn'
 
 const Login = () => {
+
+
+
+    const {isSuccess, isLoading, isError, mutate, reset} = useMutation ({
+        mutationFn: (data) => 
+            axios.post('https://hi-clist-be.vercel.app/api/user/login', data)
+            .then(response => response.data),
+            onSuccess: () => {
+                console.log('success');
+            },
+            
+    })
+
+    const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const form = event.target as HTMLFormElement;
+        const emailInput = form.querySelector('[name="username"]') as HTMLInputElement;
+        const passwordInput = form.querySelector('[name="password"]') as HTMLInputElement;
+
+        const LogInCredentials = {
+            email: emailInput.value,
+            password:passwordInput.value,
+        }
+        mutate(LogInCredentials); //this will trigger the useMutation.
+    }
+
+  
   return (
     <>
+     {isSuccess ? <LoggedIn /> :
+     <section>
```

Comment:
pwede pa ni ma improve @ArtemisPeter  like:

`<> 
       {isSuccess? <userDashBoard /> :  <loginDialog />}
   </>
  `
  
  it's a good practice to name variables and functions according to their intention/purpose.

Reply:
noted sir!